src/content/ja/updates/posts/2016/06/css-containment.markdown

```diff
@@ -0,0 +1,148 @@
+---
+title: "Chrome 52 に CSS Containment が導入"
+description: "CSS Containment という新しい仕様によって、ブラウザが行うスタイル、レイアウト、描画の範囲を制限できるようになります。"
+authors:
```

Comment:
This is unnecessary because this `authors` definition is inherited from the original en file, I guess.